WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListActivity.kt

```diff
@@ -408,12 +408,12 @@ class PostsListActivity : LocaleAwareActivity(),
                     SnackbarItem(
                             SnackbarItem.Info(
                                 view = parent,
-                                textRes = UiStringRes(holder.messageRes),
+                                textRes = holder.message as UiStringRes,
```

Comment:
I believe this manual cast is unnecessary -> moreover, it can result in an exception. Wdyt?

Reply:
Removed here a5e22f7